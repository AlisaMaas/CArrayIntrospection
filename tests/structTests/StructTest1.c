/**
* Access to a struct containing two 
* different kinds of struct fields.
**/

struct foo {
	int a;
	double b;
	struct foo *c;
};

struct bar {
	struct foo first;
	int x;
	double y;
	struct bar *z;
};

void accessBar(struct bar b) {
	b.x = 5;
	b.y = 4.5;
	b.z = &b; 
}