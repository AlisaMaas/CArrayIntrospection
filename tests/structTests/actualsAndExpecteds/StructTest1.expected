struct.bar:
	element 1 accessed
	element 2 accessed
	element 3 accessed
